airflow/contrib/hooks/redis_cluster_hook.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from rediscluster import StrictRedisCluster
+from airflow.exceptions import AirflowException
+from airflow.hooks.base_hook import BaseHook
+from airflow.utils.log.logging_mixin import LoggingMixin
+
+
+class RedisClusterHook(BaseHook, LoggingMixin):
+    """
+    Hook to interact with Redis Cluster
+
+    Note: If the Redis Cluster is launched in AWS Elasticache,
+    the hook must be initialized with skip_full_coverage_check=True option.
+    This is because AWS disallowed the CONFIG command in the Redis Cluster.
+    """
```

Comment:
We shouldn't have a doc string on the class and on the __init__ function -- only one will be displayed. Please merge this text in to the __init__ docstring.